presto-cache/src/main/java/com/facebook/presto/cache/alluxio/AlluxioCachingFileSystem.java

```diff
@@ -13,41 +13,23 @@
  */
 package com.facebook.presto.cache.alluxio;
 
-import alluxio.AlluxioURI;
-import alluxio.client.file.FileInStream;
-import alluxio.client.file.URIStatus;
-import alluxio.client.file.cache.LocalCacheFileSystem;
-import alluxio.conf.AlluxioProperties;
-import alluxio.conf.InstancedConfiguration;
-import alluxio.conf.PropertyKey;
-import alluxio.conf.Source;
-import alluxio.grpc.OpenFilePOptions;
-import alluxio.hadoop.AbstractFileSystem;
-import alluxio.hadoop.HadoopConfigurationUtils;
-import alluxio.metrics.MetricsConfig;
-import alluxio.metrics.MetricsSystem;
-import alluxio.util.ConfigurationUtils;
+import alluxio.hadoop.LocalCacheFileSystem;
 import com.facebook.presto.cache.CachingFileSystem;
 import com.facebook.presto.hive.HiveFileContext;
 import com.facebook.presto.hive.filesystem.ExtendedFileSystem;
-import com.google.common.collect.ImmutableMap;
 import org.apache.hadoop.conf.Configuration;
 import org.apache.hadoop.fs.FSDataInputStream;
+import org.apache.hadoop.fs.FileStatus;
 import org.apache.hadoop.fs.Path;
 
 import java.io.IOException;
 import java.net.URI;
-import java.util.Map;
-import java.util.Optional;
-import java.util.Properties;
-
-import static java.util.Objects.requireNonNull;
 
 public class AlluxioCachingFileSystem
         extends CachingFileSystem
 {
     private final boolean cacheValidationEnabled;
-    private AlluxioCachingFileSystemInternal cachingFileSystem;
+    private LocalCacheFileSystem cachingFileSystem;
```

Comment:
Can we rename this to localCacheFileSystem?

Reply:
done